aiohttp/client_reqrep.py

```diff
@@ -522,19 +523,18 @@ def update_body_from_data(self, body: Any) -> None:
         self.body = body
 
         # enable chunked encoding if needed
-        if not self.chunked:
-            if hdrs.CONTENT_LENGTH not in self.headers:
-                size = body.size
-                if size is None:
-                    self.chunked = True
-                else:
-                    if hdrs.CONTENT_LENGTH not in self.headers:
```

Comment:
This check was done above so it would always be true, which allows this code block to be condensed